cmake/Modules/FindTPLCUDA.cmake

```diff
@@ -1,11 +1,11 @@
 
 IF (KOKKOS_CXX_COMPILER_ID STREQUAL Clang)
    # Note: "stubs" suffix allows CMake to find the dummy
-   # libcuda.so provided by the nVidia CUDA Toolkit for
+   # libcuda.so provided by the NVIDIA CUDA Toolkit for
    # cross-compiling CUDA on a host without a GPU.
    KOKKOS_FIND_IMPORTED(CUDA INTERFACE
     LIBRARIES cudart cuda
-    LIBRARY_PATHS ENV LD_LIBRARY_PATH ENV CUDA_PATH
+    LIBRARY_PATHS ENV LD_LIBRARY_PATH ENV CUDA_PATH /usr/local/cuda
```

Comment:
Based on the Docker change below - is the library expected to be in `/usr/local/cuda/lib`? Or `/usr/local/cuda/targets/x86_64-linux/lib`?  My understanding of `find_library` behavior is that this would not locate `/usr/local/cuda/targets/x86_64-linux/lib/libcudart.so`.

Reply:
the library is in `/usr/local/cuda-X.Y/targets/x86_64-linux/lib` but there are symlinks and `/usr/local/lib64` resolves to it.